### PR TITLE
enable a rich text presentation in the setup wizard

### LINKED ISSUE #428: StepRepresentationBrowse clickable URL
Allow to click a URL in the StepRepresentationBrowse.

### PR DESCRIPTION
**Pull request by plata**
Now, e.g. links can actually be links and not just plain text.

fixes #428 

rich text presentation

### CODE REVIEW

**Comment by qparis** [comment]
Great Idea. I think it would ve preferable to create a generic html step

**Comment by plata** [comment]
Just trying this. I think it will make a problem with the CLI.

**Comment by qparis** [comment]
For CLI, we can strip tags

**Comment by plata** [comment]
Is there any way to do that automatically or do I have to list them all?

File: phoenicis-scripts/src/main/java/org/phoenicis/scripts/wizard/SetupWizard.java [hunk 1]
```diff
@@ -118,6 +118,16 @@ public Void message(String textToShow) {
     }
 
     /**
+     * Show a free script presentation
+     *
+     * @param htmlToShow the free presentation HTML to showRightView
+     */
+    @Override
+    public Void presentationHtml(String htmlToShow) {
+        return messageSender.runAndWait(message -> setupWindow.showPresentationStep(message, htmlToShow));
```
Comment (qparis) [inline phoenicis-scripts/src/main/java/org/phoenicis/scripts/wizard/SetupWizard.java]: What is the difference between message and htmlToShow?

Comment (plata) [inline phoenicis-scripts/src/main/java/org/phoenicis/scripts/wizard/SetupWizard.java]: message is the callback. It's the same as any other method in this class.

**Comment by qparis** [comment]
My mistake

**Pull request merged**